src/obb_rules/serializer/reader.cljc

```diff
@@ -0,0 +1,161 @@
+(ns ^{:added "2.1.0"
+      :author "Pedro Santos"}
+  obb-rules.serializer.reader
+  "Loads and processes a game from a serialized text format"
+  (:require [clojure.string :as string]
+            [obb-rules.serializer.common :as common]
+            [obb-rules.game :as game]
+            [obb-rules.result :as result]
+            [obb-rules.board :as board]
+            [obb-rules.turn :as turn]
+            [obb-rules.host-dependent :as host]
+            [obb-rules.game-mode :as game-mode]))
+
+(defmulti str->action
+  "Converts a action in string format to a raw action"
+  (fn [s]
+    (first s)))
+
+(defn get-int
+  "Given a string, gets the char on the specific position, and returns it
+  as an integer"
+  [s pos]
+  (host/parse-int (str (nth s pos))))
```

Comment:
This is really a nitpick, but couldn't this be made private? I'd also move the `defmulti` definition next to the concrete implementations.